src/interfaces/jira.ts

```diff
@@ -86,17 +86,29 @@ export interface JiraAssociation {
 }
 
 export interface JiraRemoteLinkData {
-	product: string;
 	remoteLinks: JiraRemoteLink[];
 }
 
+interface IssueKeysAssociation {
```

Comment:
/nit change name to `Association` or `JiraAssociation` to match api schema?

Reply:
Resolved with merge conflict